### PR TITLE
Payme

### PR DESCRIPTION
**Pull request by Torrib**

### CODE REVIEW

File: apps/authentication/models.py [hunk 1]
```diff
@@ -113,6 +113,7 @@ class OnlineUser(AbstractUser):
     linkedin = models.URLField(_(u"linkedin"), blank=True, null=True)
     gender = models.CharField(_(u"kjønn"), max_length=10, choices=GENDER_CHOICES, default="male")
     bio = models.TextField(_(u"bio"), blank=True)
+    saldo = models.PositiveSmallIntegerField(_(u"saldo"), default=0)
```
Comment (hernil) [inline apps/authentication/models.py]: It's not unheard of not having an integer amount of money in an account. Maybe reconsider field type?

File: apps/inventory/dashboard/urls.py [hunk 2]
```diff
@@ -10,9 +10,6 @@
     url(r'^item/(?P<item_pk>\d+)/delete/$', 'item_delete', name='dashboard_inventory_delete'),
     url(r'^item/(?P<item_pk>\d+)/batch/new/$', 'batch_new', name='dashboard_inventory_batch_new'),
     url(r'^item/(?P<item_pk>\d+)/batch/(?P<batch_pk>\d+)/$', 'batch', name='dashboard_inventory_batch'),
-    url(
-        r'^item/(?P<item_pk>\d+)/batch/(?P<batch_pk>\d+)/delete/$',
-        'batch_delete',
-        name='dashboard_inventory_batch_delete'
-    ),
+    url(r'^item/(?P<item_pk>\d+)/batch/(?P<batch_pk>\d+)/delete/$', 'batch_delete', name='dashboard_inventory_batch_delete'),
```
Comment (hernil) [inline apps/inventory/dashboard/urls.py]: Will fail on line length (max 120)

File: apps/inventory/models.py [hunk 3]
```diff
@@ -4,10 +4,21 @@
...
+    name = models.CharField(_(u"Kategori"), max_length=50)
+
 
 class Item(models.Model):
 
     name = models.CharField(_(u"Varetype"), max_length=50)
+    description = models.CharField(_(u"Beskrivelse"), max_length=50, null=True, blank=True)
+    price = models.IntegerField(_(u"Pris"), null=True, blank=True)
+    available = models.BooleanField(_(u"Til salgs"), default=False)
+    category = models.ForeignKey(ItemCategory, verbose_name=_(u"Kategori"), related_name="category", null=True, blank=True)
```
Comment (hernil) [inline apps/inventory/models.py]: Double check length here as well

File: apps/shop/migrations/0001_initial.py [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Order',
+            fields=[
+                ('id', models.AutoField(verbose_name='ID', serialize=False, auto_created=True, primary_key=True)),
+                ('object_id', models.PositiveIntegerField()),
+                ('price', models.DecimalField(max_digits=10, decimal_places=2, blank=True)),
+                ('quantity', models.PositiveIntegerField(default=1, validators=[django.core.validators.MinValueValidator(1)])),
```
Comment (hernil) [inline apps/shop/migrations/0001_initial.py]: Length

**Comment by sklirg** [comment]
I'm pretty sure we don't run flake8 on migrations, fwiw.

**Comment by cXhristian** [comment]
Where do you see flake8 changes om migrations?

**Comment by sklirg** [comment]
I only read the email I got, where @hernil commented on `apps/shop/migrations/0001_initial.py`. I guess he realized and deleted the comment. :sunglasses: 
screenshot 2016-01-28 15 19 03

**Pull request merged**